cmd/auth/main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"net/http"
+	"os"
+
+	"github.com/thatnerdjosh/example-webservices/internal/core"
+)
+
+const (
+	defaultPort string = "8080"
+)
+
+func main() {
```

Comment:
The idea here is that it would reach to an auth service for authentication (which for now is just a simple stub). That image will be separated out (as per the TODO in the Dockerfile), definitely not ideal switching on the entry point like I'm doing, short term solution.